graphics/grafx2/Portfile

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                grafx2
+version             2.5
+categories          graphics
+maintainers         @miniupnp
```

Comment:
It is best if you include both your github handle and email address (see the guide for details on the format), and if you have no specific reason to prohibit team members from making small changes without consulting you, you should set your ports "openmaintainer" as well.